app/models/concerns/scholars_archive/to_solr_behavior.rb

```diff
@@ -0,0 +1,25 @@
+module ScholarsArchive
+  module ToSolrBehavior
```

Comment:
Yeah I like this. to_solr is a little indirect and I wouldnt quite know what it does. But that name describes what the to_solr method override is doing. 👍